### ISSUE TITLE
Feature Request: Filter Apps

### ISSUE THREAD

**Issue by qoomon** [EthanSK/Menu-Bar-Dock#22]
Would be great to have the option to select wich running apps should be shown in menu bar.

**Comment by wolveix** [comment]
I suppose one implementation option for this would be to an "Ignored Apps" user preference, similar to the "Regular Apps" preference

**Comment by fizzz71** [comment]
Yes, i appreciate that application a lot. 

i'm using it as task bar to switch between apps, and secondly, to start my most important apps

Only thing missing for me, is to be able to hide selected apps (like mamp or other), that i don't want to appear.

Is it planned to do it?


@EthanSK Bravo for this app, that's what i was looking for. The don't really like the Apple Dock.

I would be glad to contribute

I installed xcode, etc...  on a macbook air m1, but i can't build, always error with PODS things (maybe , it's because i'm on m1?)  Any idea on that?

Thanks

**Comment by EthanSK** [comment]
> I suppose one implementation option for this would be to an "Ignored Apps" user preference, similar to the "Regular Apps" preference

true, maybe a segment control that switches the regular apps table to running apps filter, or maybe a button that opens a new window with the table to add apps to the filter. 

I'm quite busy atm but if someone wants to submit a PR i'm happy to approve

**Comment by wolveix** [comment]
Yeah, I didn't get around to implmenting it myself as I didn't need it, and couldn't quite figure out an elegant way to move both "Regular Apps" and "Ignored Apps" into their own pop-up window (likely due to my lack of experience working within XCode)

**Comment by charlielewisme** [comment]
+1 for the 'ignored apps' list FR
